modules/objectdb_profiler/editor/snapshot_data.cpp

```diff
@@ -198,15 +198,15 @@ HashSet<ObjectID> SnapshotDataObject::_unique_references(const HashMap<String, O
 		obj_set.insert(pair.value);
 	}
 
-	return obj_set;
+	return HashSet<ObjectID>(obj_set);
 }
 
 HashSet<ObjectID> SnapshotDataObject::get_unique_outbound_refernces() {
-	return _unique_references(outbound_references);
+	return HashSet<ObjectID>(_unique_references(outbound_references));
```

Comment:
This one introduces a copy, replacing a move

```suggestion
	return _unique_references(outbound_references);
```